layouts/shortcodes/studio-hub.html

```
@@ -0,0 +1,259 @@
+    {{ if eq $kind "main" }}
+    <section class="authz-hero">
+        <div class="authz-wrap authz-hero-grid">
+            <div>
+                <div class="authz-eyebrow">Altinn Studio</div>
+                <h1>{{ if $nb }}Bygg dine tjenester med Altinn Studio{{ else }}Build your services using Altinn Studio{{
+                    end }}</h1>
+                <p class="authz-lead">{{ if $nb }}Alt du trenger for å utvikle, publisere og forvalte digitale tjenester
+                    for innbyggere og næringsliv.{{ else }}Everything you need to develop, publish and manage digital
+                    services for citizens and businesses.{{ end }}</p>
+                <div class="authz-actions">
+                    <a class="authz-btn authz-btn-primary" href="{{ "altinn-studio/v9/getting-started/" | relLangURL
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)